pkg/plugins/golang/v4/scaffolds/internal/templates/api/types.go

```diff
@@ -64,6 +64,7 @@ func (f *Types) SetTemplateDefaults() error {
 //nolint:lll
 const typesTemplate = `{{ .Boilerplate }}
 
+// +kubebuilder:ac:generate=true
```

Comment:
I think the feature must be optional.
We should not promote it by default, right?
By default, people should still be working as they do today.

Reply:
I don't have a project that uses this configuration, however I ran the generate testdata using it: https://github.com/kubernetes-sigs/kubebuilder/commit/537f54ef5a525ebf955862280d602bd0b9fe71d7